authorization/auth.js

```diff
@@ -0,0 +1,22 @@
+const passport = require("passport");
+
+const auth = (req, res, next) => {
+  passport.authenticate("jwt", { session: false }, (err, user) => {
+    if (!user || err) {
```

Comment:
tu trzeba dodać sprawdzanie czy user.token jest taki sam jak wysyłasz w nagłówku (req.headers) tylko przy sprawdzaniu musisz odjąć to "bearer " z przodu. Bo tak to mogę dać logout a /current na starym tokenie nadal działa